js/viz.js

```diff
@@ -105,12 +105,18 @@ const viz = {
     for (const d of this.nodes) {
       this.context.beginPath();
       this.context.moveTo(d.x, d.y);
-      this.context.arc(d.x, d.y, this.circleRadius, 0, 2 * Math.PI);
       if (d.firstParty) {
-        this.context.fillStyle = 'red';
+        this.context.arc(d.x, d.y, this.circleRadius, 0, 2 * Math.PI);
       } else {
-        this.context.fillStyle = 'blue';
+        // draw third party as equilateral triangle
```

Comment:
@biancadanforth please put this into a new function :) `drawFirstParty(x,y)` and `drawThirdParty(x,y)`